charts/monitoring/templates/prometheusrules.yaml

```diff
@@ -11,7 +11,7 @@ spec:
     - alert: {{ .Values.name }}-datadrift-alert
       annotations:
         message: '{{ .Values.name }} has data drift...'
```

Comment:
```suggestion
        message: '{{ .Values.name }} - Data drift detected.'
```

Reply:
actually, what would be a better message here @RHRolun 😅

Reply:
Message updated, lmk what you think :)

Reply:
awesome! thanks Robert!